backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserPermission.java

```diff
@@ -2,12 +2,16 @@
 
 /**
  * Permissions that a user may hold, usually via their {@link OrganizationRole}.
+ *
+ * DO: Keep updated with UserPermission in schema.graphqls
```

Comment:
Though if you don't, it'll fail startup with a reasonably informative message. 😺